_data/projects/keep.yml

```
@@ -0,0 +1,12 @@
+name: Keep
+desc: Open-source alert management and workflow automation platform.
+site: https://www.keephq.dev
+tags:
+  - monitoring
+  - alerting
+  - alerts
+  - workflow-automation
+  - automation
+upforgrabs:
+  name: good first issues
+  link: https://github.com/keephq/keep/issues?q=label%3A%22good%20first%20issue%22
```

:art: a simpler URL that's equivalent to the current value

```suggestion
  link: https://github.com/keephq/keep/labels/good%20first%20issue
```

---

Gotcha, thanks!